centralizedmetrics/providers/mixpanel.go

```diff
@@ -0,0 +1,190 @@
+package providers
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/status-im/status-go/centralizedmetrics/common"
+)
+
+const MixpanelBaseURL = "https://api.mixpanel.com"
+
+var MixpanelToken = ""
+var MixpanelAppID = ""
+
+// MixpanelMetricProcessor implements MetricProcessor for Mixpanel
+type MixpanelMetricProcessor struct {
+	AppID   string
+	Secret  string
+	BaseURL string
+}
+
+// NewMixpanelMetricProcessor is a constructor for MixpanelMetricProcessor
+func NewMixpanelMetricProcessor(appID, secret, baseURL string) *MixpanelMetricProcessor {
+	return &MixpanelMetricProcessor{
+		AppID:   appID,
+		Secret:  secret,
+		BaseURL: baseURL,
+	}
+}
+
+func (amp *MixpanelMetricProcessor) GetAppID() string {
+	if len(amp.AppID) != 0 {
+		return amp.AppID
+	}
+	return MixpanelAppID
+}
+
+func (amp *MixpanelMetricProcessor) GetToken() string {
```

Comment:
It's probably `mmp`? 
Or maybe just keep `p` (Processor) for both implementations?